backend/database/migrations/sqls/20200627203545-seed-up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE Users
```

Comment:
@marcianszlovak separate this into:
- create table migration
- insert example data
use the javascript wrappers instead